### ISSUE TITLE
LAST DITCH RESORT (ONLY DO IN FREE TIME) MANUAL PID CONTROLLER

### ISSUE THREAD

**Issue by Cybertron51** [troy-robosub/robosub2024#3]
Ok so basically rn we got this below command working:

def manualControl(x, y, z, r):
    master.mav.manual_control_send(
        master.target_system,
        x,  # pitch for sub, back/forward on joystick [-1000,1000], respectively
        y,  # roll for sub, left/right on joystick [-1000,1000], respectively
        z,  # thrust for sub, slider on joystick [0,1000]
        r,  # yaw for sub, clockwise/counterclockwise on joystick [-1000,1000], respectively
        0)  # buttons

the variables themselves are a bit off, these are the actual commands:
manualControl(750,0,500,0) to go forward
manualControl(0,0,500,300) for clockwise (havent verified, it might be counterclockwise)

basically, you can use the command getHeading() to get the initial heading before doing any movement, and then once it starts moving, once you detect an error with the heading, change the last argument in manualControl to re-adjust for the change

can get fancy with pid equation but would 1000% recommend getting it to work on a basic level first
